embedding/csharp/Tizen.Flutter.Embedding/FlutterServiceApplication.cs

```diff
@@ -20,20 +19,15 @@ public class FlutterServiceApplication : ServiceApplication, IPluginRegistry
         public string DartEntrypoint { get; set; } = string.Empty;
 
         /// <summary>
-        /// The list of Dart entrypoint arguments.
+        /// Whether the app has started.
         /// </summary>
-        private List<string> DartEntrypointArgs { get; } = new List<string>();
```

Comment:
I have a question. @swift-kim 
Application doesn't originally have a scenario to set Dart engtrypoint argument?
Couldn't it be opened to change the private property to public and set it the same as DartEntryPoint?

Reply:
As we decided not to implement the entrypoint arguments in https://github.com/flutter-tizen/flutter-tizen/pull/387, the property seems no longer necessary. If you want to allow manually passing the arguments from the runner code (I don't think there will be such use case), we can make the property public.